src/cmd/split.h

```diff
@@ -91,69 +92,76 @@ char* strdup_with_new(const char* the_string) {
 // See //util/csv/parser.h for more complete documentation.
 //
 // ----------------------------------------------------------------------
-void SplitCSVLineWithDelimiter(char* line, char delimiter, std::vector<char*>* cols) {
+void SplitLineWithDelimiter(char* line, const char* delimiter, std::vector<char*>* cols, const char enclosed) {
     char* end_of_line = line + strlen(line);
     char* end;
     char* start;
+    int delimiter_len = strlen(delimiter);
 
-    for (; line < end_of_line; line++) {
-        // Skip leading whitespace, unless said whitespace is the delimiter.
-        while (absl::ascii_isspace(*line) && *line != delimiter) ++line;
+    for (; line < end_of_line; line += delimiter_len) {
+        // Skip leading whitespace, unless said whitespace is the part of delimiter.
+        while (absl::ascii_isspace(*line) && *line != delimiter[0]) ++line;
```

Comment:
if delimter is empty, it will coredump here.
prevent it.

Reply:
And if delimeter is space, we will act in a strange way. We should disable space delimiter, or check it before `while(isspace)`

Reply:
Resolved